### DISCUSSION TITLE
Paid Events

### DISCUSSION THREAD

**Discussion by paragonn** [solspace/craft-calendar#239] [Questions & Answers] [answered]
Is it possible for users to register and pay for events created in the Calendar plugin?

**Comment by kjmartens** [comment, marked as answer]
Sorry for the delay, @paragonn.

This currently isn't possible with Calendar. You could, however, attempt to use Freeform and its Payments feature to pair a form with a Calendar event and pass along a $ value, etc. However, this will definitely have some limitations, such as not being able to allow multiple tickets, limiting number of tickets available, and having different price points. 🙂

**Comment by paragonn** [comment]
Thanks for the response @kjmartens
